### PR TITLE
Introduce bug,feature request and support templates

### PR DESCRIPTION
**Pull request by tkatila**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System (please complete the following information):**
+ - OS: [e.g. Ubuntu]
+ - Version [e.g. 22.04]:
+ - Kernel [e.g. 5.15]:
+ - Hardware info [e.g. SPR with QAT]:
```
Comment (eero-t) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Consistent use of version & double-colon, request also device plugins version:
```suggestion
 - OS version: [e.g. Ubuntu 22.04]
 - Kernel version: [e.g. Linux 5.15]
 - Device plugins version: [e.g. v0.29.0]
 - Hardware info: [e.g. SPR with QAT]
```

**Review by eero-t — CHANGES_REQUESTED** [review]
"A clear and concise" is repeated a lot though, maybe just one of the adjectives would be enough?

File: .github/ISSUE_TEMPLATE/support.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+<!-- BEFORE SUBMISSION: Please read any known issues and workarounds in the plugin's readme. -->
+
+**Describe the support request**
+A clear and concise description of what you are looking support for.
+
+**System (please complete the following information if applicable):**
+ - OS version: [e.g. Ubuntu 22.04]
+ - Kernel version: [e.g. Linux 5.15]
```
Comment (eero-t) [inline .github/ISSUE_TEMPLATE/support.md:14]: Device plugins version may be relevant here too:
```suggestion
 - Kernel version: [e.g. Linux 5.15]
  - Device plugins version: [e.g. v0.29.0]
```

**Review by eero-t — DISMISSED** [review]
Support requests may need device plugins version too, but otherwise texts look fine.

**Comment by tkatila** [comment]
Amended.

fyi, checks won't run on this PR as it's only .md changes.

**Review by mythi — APPROVED** [review]

**Pull request merged**